4-storage/storage-loader/lib/snowplow-storage-loader/redshift_loader.rb

```diff
@@ -133,17 +133,21 @@ def get_shredded_statements(config, target, s3)
           schema = extract_schema(target[:table])
 
           ShreddedType.discover_shredded_types(s3, config[:aws][:s3][:buckets][:shredded][:good], schema).map { |st|
-
-            jsonpaths_file = st.discover_jsonpaths_file(s3, config[:aws][:s3][:buckets][:jsonpath_assets])
-            if jsonpaths_file.nil?
-              raise DatabaseLoadError, "Cannot find JSON Paths file to load #{st.s3_objectpath} into #{st.table}"
+            if config[:skipshred].include?(st.table)
+              #actions to trigger while escaping the table shreding
+		          SqlStatements.new("ANALYZE atomic.events")
```

Comment:
A couple of things here:

* We don't automatically assume that the table is called "atomic.events" - the user can set the name of the table in the configuration file
* It looks like this will run `ANALYZE atomic.events` once per skipped shredded type. We only want to run it once.

I don't think we actually need to create any ANALYZE statements here, because they are already created [here](https://github.com/snowplow/snowplow/blob/b772f6e089bb9e0b3bdbed93a4ab18902c4f1a4a/4-storage/storage-loader/lib/snowplow-storage-loader/redshift_loader.rb#L108). So for skipped types we don't need to add anything to the array returned by get_shredded_statements. One way to do this would be to return `nil` to the array of SqlStatements for each skipped type, then call `.compact` on the resulting array to remove the `nil`s.